imagevector/images.yaml

```diff
@@ -2,11 +2,11 @@ images:
 # registry cache StatefulSet
 - name: registry
   sourceRepository: github.com/distribution/distribution
-  repository: eu.gcr.io/gardener-project/3rd/registry
+  repository: europe-docker.pkg.dev/gardener-project/releases/3rd/registry
   tag: "2.8.3"
 # registry-configuration-cleaner DaemonSet
 - name: alpine
-  repository: eu.gcr.io/gardener-project/3rd/alpine
+  repository: europe-docker.pkg.dev/gardener-project/releases/3rd/alpine
   tag: "3.18.4"
```

Comment:
How these images got available under `europe-docker.pkg.dev`? These images are being copied to GCR with a prow job - see https://github.com/gardener/ci-infra/blob/master/config/images/images.yaml and https://github.com/gardener/ci-infra/tree/master/config/images.
I think first this prow job has to be adapted to maintain the copied images in AR and not in GCR.

cc @oliver-goetz

Reply:
@ialidzhikov I copied this specific version manually. considering there is already a replication mechanism in place, I agree this ought to be adjusted so future versions will also be considered. How is metadata about updated alpine-versions currently being transported e.g. into this location?

Reply:
alpine is also maintained by the same mechanism: https://github.com/gardener/ci-infra/blob/8ee30aa71c09d66d0416a18048a6ab85a593cc03/config/images/images.yaml#L26-L31.

I don't think copying manually is a sustainable approach. We first need to adjust the above-described prow job to copy to AR.

Reply:
yes, of course. I just copied this one version as a one-off. let's wait for @oliver-goetz then

Reply:
@ialidzhikov : I adjusted image-copying [here](https://github.com/gardener/ci-infra/pull/1066)